.storybook/config.js

```diff
@@ -25,12 +27,17 @@ function loadStories() {
 const context = require.context('../components', true, /\.twig$/)
 context.keys().forEach(key => {
   var template = context(key);
+  const rootPath = path.join(__dirname);
   Twig.twig({
+    base: rootPath, // Possibly can set a base directory for everything to work from?
+    namespaces: {
+      'atoms': 'components/01-atoms',
+    },
     id: key,
     data: template.tokens,
-    allowInlineIncludes: true,
+    // allowInlineIncludes: true, // This seems to be an alternative to namespacing which allows relative paths.
     rethrow: true
-  });
+  }).render();
```

Comment:
I'm not sure if `render()` is needed here - it was just something from the example code I tried to get things working.